game-settings.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>GameSelection</title>
+      <link rel="stylesheet" href="styles/style.css">
+      <script src="https://kit.fontawesome.com/84576b974f.js" crossorigin="anonymous"></script>
+   </head>
+   <body class="page">
+      <div class="menu">
+         <img class="menu__logo" src="img/StarWarsLogo.png" alt="">
+         <nav class="menu__list">
+            <a class="menu__item menu__item--selected" href="#">People</a>
+            <a class="menu__item menu__item--unselected" href="#">Vehicles</a>
+            <a class="menu__item menu__item--unselected" href="#">Starships</a>
+         </nav>
+      </div>
+      <div class="settings">
+         <form>
+            <label for="ApiKey">Google Vision API Key:</label>
+            <input class="ApiKey__input" type="text" id="ApiKey" name="ApiKey">
```

Comment:
chyba spoko trzymać się jakiejś konwencji - proponuję api_key albo apiKey, bo ApiKey to nawet nie camel case i kojarzy i mi się z konwencją zarezerwowaną dla klas

Reply:
poprawione